src/modules/asab-config/ConfigContainers/ConfigContainer.js

```diff
@@ -13,60 +13,204 @@ import { useTranslation } from 'react-i18next';
 import { TreeViewComponent } from "./TreeViewComponent";
 import ConfigEditor from "./ConfigEditor";
 import ConfigList from "./ConfigList";
-// import reducer from './reducer';
+import ConfigImport from "./ConfigImport";
 
 function ConfigContainer(props) {
 
-	const { t, i18n } = useTranslation();
+	const App = props.app;
+	const ASABConfigAPI = App.axiosCreate('asab_config');
+	const serviceURL = App.getServiceURL('asab_config');
+	const { t } = useTranslation();
 
 	const configType = props.match.params.configType;
 	const configName = props.match.params.configName;
 
-	const [ createConfig, setCreateConfig ] = useState(false);
-
 	const homeScreenImg = props.app.Config.get('brand_image').full;
 	const homeScreenAlt = props.app.Config.get('title');
 
+	const [ treeData, setTreeData ] = useState({});
+	const [ createConfig, setCreateConfig ] = useState(false);
+	const [ chosenPanel, setChosenPanel ] = useState("configurator");
+	const [ typeList, setTypeList ] = useState([]);
+	const [ treeList, setTreeList ] = useState({});
+	const [ openNodes, setOpenNodes ] = useState([]); // Set open nodes in the TreeMenu
+
+
+	// To get the full overview on schemas and configs it is needed to update the tree list and data state
+	useEffect(() => {
+		getTypes();
+	}, []);
+
+	useEffect(() => {
+		if (typeList.length > 0) {
+			getTree();
+		}
+	}, [typeList])
+
+	useEffect(() => {
+		getChart();
+	}, [treeList]);
+
+
+	// Obtain list of types
+	// TODO: add Error Card screen when no types are fetched
+	const getTypes = async () => {
+		try {
+			let response = await ASABConfigAPI.get("/type");
+			if (response.data.result != 'OK') {
+				throw new Error("Unable to get data for tree menu");
+			}
+			// Sort data
+			let sortedData = response.data.data;
+			sortedData = sortedData.sort();
+			setTypeList(sortedData);
+			// TODO: validate responses which are not 200
+		}
+		catch(e) {
+			console.error(e);
+			App.addAlert("warning", t(`ASABConfig|Unable to get data for tree menu`));
```

Comment:
@Aringoaway Its better to use `props.app.addAlert(...)` that we know the full path from where it is taken